piqe_ocp_lib/tests/resources/test_ocp_prometheus_client.py

```diff
@@ -0,0 +1,36 @@
+import logging
+import pytest
+from piqe_ocp_lib.api.resources.ocp_prometheus_client import OcpPrometheusClient
+from piqe_ocp_lib import __loggername__
+
+logger = logging.getLogger(__loggername__)
+
+
+@pytest.fixture(scope="session")
+def ocp_prom_client(get_kubeconfig):
```

Comment:
You spell out the name prometheus everyone in the patch set except for the title of this method. Is there a reason you shortened prometheus to prom here?

Reply:
You are right...I shorten this since this is fixture name only and returns object of Prometheus client class.

Reply:
I think it would be good to name it as ocp_prometheus_client than ocp_prom_client :-)